README.md

```diff
@@ -107,6 +107,70 @@ end
 
 _Note: `drop_split` (a.k.a. `finish_split`) does not physically delete split data from mixpanel or Test Track's database._
 
+### Generating Split Migrations
+
+Split configuration changes can be generated using Rails generators that are included with the TestTrack Rails client.
+
+```
+rails generate test_track:migration add_name_of_split
```

Comment:
Yeah, that was the original approach and for some reason I thought that the flag approach might be better. I can definitely see the utility in having this act like regular rails migrations, so I'll update this feature accordingly.